mvvm-helpers/common-converters.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Converters
+page_title: Converters 
+description: This article describe built-in converter which can be used.
+slug: common-converters
+tags: converter
+published: True
+position: 2
+---
+
+# Converters
+
+Telerik controls provide a set of different built-in __IValueConverters__ which can be used to perform conversion between different types.
+
+>To get access to these converters you can add reference to the __Telerik.Windows.Controls.dll__.
+
+* [BooleanToOpacityConverter](#booleantoopacityconverter)
+* [InvertedBooleanToOpacityConverter](#invertedbooleantoopacityconverter)
+* [BooleanToVisibilityConverter](#booleantovisibilityconverter)
+* [InvertedBooleanToVisibilityConverter](#invertedbooleantovisibilityconverter)
+* [EnumToVisibilityConverter](#enumtovisibilityconverter)
+* [ColorToBrushConverter](#colortobrushconverter)
+* [NullToVisibilityConverter](#nulltovisibilityconverter)
+* [NumberToVisibilityConverter](#numbertovisibilityconverter)
+* [DateTimeTypeConverter](#datetimetypeconverter)
+* [UppercaseConverter](#uppercaseconverter)
+* [BinaryImageConverter](#binaryimageconverter)
+* [DoubleToThicknessConverter](#doubletothicknessconverter)
+* [OpacityConverter](#opacityconverter)
+
+
```

Comment:
Here you can add a code snippet where all converters are defined in the Resources collection of an element. I think this would be very useful for lazy arse developers like me which like copying the definition of the convert and pasting it in their app. 

An example:
    <FrameworkElement.Resources>
       <telerik:BooleanToVisibilityConverter x:Key="BooleanToVisibilityConverter"/>
       <telerik:InverterBooleanToVisibilityConverter x:Key="InverterBooleanToVisibilityConverter"/>
       etc.
    </FrameworkElement.Resources>

Reply:
Note that I use FrameworkElement because it is more abstract and can be set on every FrameworkElement. You can use this approach to avoid defining two different snippets for two different elements. For example, if you want to show that you can add the converter in the UserControl (Silverlight) and Window (WPF) - you can use FrameworkElemetn instead. This way you will have a single code snippet for both platforms.